controllers/routing/routing_fixtures_test.go

```diff
@@ -0,0 +1,62 @@
+package routing_test
+
+import (
+	"context"
+
+	. "github.com/onsi/gomega"
+	"github.com/opendatahub-io/odh-platform/pkg/cluster"
+	"github.com/opendatahub-io/odh-platform/pkg/metadata"
+	"github.com/opendatahub-io/odh-platform/test"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func getClusterDomain(ctx context.Context) string {
```

Comment:
Maybe it would be handy to pass client next to the ctx and by doing so making this func not coupled with the envTest and its `.Client`.

Reply:
[538234e](https://github.com/maistra/odh-platform/pull/39/commits/538234edfd01f3afce213ad0fd4a80c19d5e342b) added arg here